src/AddTodoForm.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
```

Comment:
`props` represents the entire object that's being passed from parent to child. Nice job working with it. Now, what's an advanced technique we could use to directly access each property in `props`? Not only will this give us a clear display of which props are being passed along but it'll also clean up our code when we want to access each prop.

Reply:
If you can't figure this one out you can go still ahead and merge it. I approved your assignment b/c you met all the requirements. This one is slightly beyond what the assignment requires but it's a common pattern you'll encounter often. Ping me for a review before merging if you are able to refactor it.